src/Core/composer.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "knplabs/snappy-core",
+    "require": {
+        "php": ">=8.1",
```

Comment:
Since we are aiming to support Symfony 5.4 we should downgrade this requirement to `>=7.2.5`